google/cloud/telcoautomation/doc/main.dox

```diff
@@ -0,0 +1,46 @@
+/*!
+
+@mainpage Telco Automation API C++ Client Library
+
+An idiomatic C++ client library for the [Telco Automation API][cloud-service-docs], a service
+to APIs to automate 5G deployment and management of cloud infrastructure and network functions.
```

Comment:
s/, a service to/--/ ?

Reply:
PTAL.  I am almost convinced to remove this sentence altogether.